lib/setup/User.js

```diff
@@ -81,76 +81,64 @@ class User {
   /**
    * Private method which Deploys gnosis MultiSig master copy contract.
    *
-   * @param txOptions Tx options.
    * @returns {txObject} - Transaction object.
    * @private
    */
-  _deployMultiSigMasterCopyRawTx(txOptions) {
+  _deployMultiSigMasterCopyRawTx() {
```

Comment:
Please remove `txOptions` argument from where this method is called i.e. deployMultiSigMasterCopyRawTx at line `32`.
Similar comment for method `_deployTokenHolderMasterCopyRawTx` and `_deployUserWalletFactoryRawTx`.